### PR TITLE
Add references in History

### PR DESCRIPTION
**Pull request by MattiSG**

### CODE REVIEW

File: content/about.en.md
Comment (sandcha) [inline content/about.en.md:29]: 👍 OGP report link.

Comment (sandcha) [reply content/about.en.md:29]: The model link is misleading but the description isn't exactly true. Rephrase it?: `a small team of OpenFisca France maintainers and OpenFisca Tunisia volunteers managed to model Sénégal's income revenue tax`.

As you know, I was part of the OpenFisca Tunisia team (= @benjello and members of [Cahiers de la Liberté](https://www.facebook.com/cahiers.de.la.liberte/)). We updated a bit openfisca_tunisia (+/- taxes and contributions) and developed a web interface on to of it. 

The senegalese model was developed by OpenFisca maintainers and 2 tax inspectors from Senegal. If I remember correctly they also reused the web interface that the Tunisian team developed.

So, regarding the added link here, during OGP at least 2 models were updated/developed: openfisca-senegal and [openfisca-tunisia](https://github.com/openfisca/openfisca-tunisia) (+openfisca-france ? 🤔).

> And we can remove the accents on `Sénégal` in English :)

Comment (MattiSG) [reply content/about.en.md:29]: Thanks for these precisions @sandcha! I have to say the specific level of details that make the description “not exactly true” is a bit lost on me, and I fail to see the difference between what I wrote and what you write here. Since that seems important enough for you to not approve this PR, would you mind offering a specific suggestion for the rephrasing you would like to see here? Thank you 🙂

**Review by clementbiron — APPROVED** [review]

**Pull request merged**
